creditmanagement/templatetags/credit_tags.py

```diff
@@ -1,15 +1,14 @@
 from django import template
-from django.utils.safestring import mark_safe
+from django.utils.formats import localize
 
 register = template.Library()
 
 
 @register.filter
 def euro(value):
     """Format for euro values."""
-    amount = "{:.2f}".format(abs(value))
-    v = "{}€{}".format("-" if value < 0 else "", amount)
-    return mark_safe(v)
+    # Could surround this with `mark_safe` but is probably not necessary
```

Comment:
I see no reason why it's here either (except for the euro sign, but that should work without making it safe. After the update the comment is not neccessary though, so maybe just remove that line
```suggestion
```

Reply:
Yes good point. I'll remove it.